### PR TITLE
Add vars upstream to make downstream transition easier

### LINKED ISSUE #38: Add new server playbook vars for downstream
Add new vars to server playbook to help with downstream:
```
server_image_name: 'quipucords'
server_image_tag: ''
podman_tls_verify: true
```

### PR DESCRIPTION
**Pull request by myersCody**
closes #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,6 +159,16 @@ qpc-tools cli install --offline-files /qpc_tools/install/packages
 Note:
  - Optionally run any secret post install scripts you included in `tools_config.tar.gz`
 
+## Vagrant: Testing Advanced Args
+
+Sometimes flags that are not externally available will need to be flipped in order to bypass special case issues during SEAP engagements. For example, some customers may not be able to use supervisord. To test these advanced internal arguements do the following:
```
Comment (abaiken) [inline README.md]: change arguements - arguments

File: qpc_tools/server/ansible/install/roles/server/tasks/init_vars_online.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@
   changed_when: false
   when:
     - server_version is not defined
+    - server_image_tag == ""
```
Comment (kholdaway) [inline qpc_tools/server/ansible/install/roles/server/tasks/init_vars_online.yml]: Revert this and downstream replace file.

File: qpc_tools/cli/ansible/install/roles/cli/tasks/load_rpm_online.yml [hunk 3]
```diff
@@ -46,8 +46,14 @@
...
+- name: Set the Quipucords CLI RPM package name
+  set_fact:
+    cli_package_name: "{{ cli_github_release_url }}/qpc.el{{ ansible_distribution_major_version }}.noarch.rpm"
+  when:
+    - cli_package_name == ""
+
 - name: Install the Quipucords CLI
   yum:
-    name: '{{ cli_github_release_url }}/{{ qpc_package_name }}'
+    name: '{{ cli_package_name }}'
```
Comment (kholdaway) [inline qpc_tools/cli/ansible/install/roles/cli/tasks/load_rpm_online.yml]: Revert this and replace this whole file downstream with a small role that does a simple yum install.

File: qpc_tools/cli/ansible/install/roles/cli/tasks/init_vars_offline.yml [hunk 4]
```diff
@@ -1,12 +1,18 @@
 ---
 
+- name: Set the Quipucords CLI RPM package name
```
Comment (kholdaway) [inline qpc_tools/cli/ansible/install/roles/cli/tasks/init_vars_offline.yml]: Move this back to init_vars.yml

**Review by kholdaway — CHANGES_REQUESTED** [review]

**Review by abaiken — APPROVED** [review]
Looks good 😄

File: qpc_tools/cli/install.py [hunk 5]
```diff
@@ -58,6 +58,7 @@ def __init__(self, subparsers):
                                  help=_(messages.CLI_INSTALL_PORT_HELP),
                                  required=False)
         self.parser.add_argument('--advanced', dest='server_advanced',
+                                 nargs='+', default=[],
```
Comment (abaiken) [inline qpc_tools/cli/install.py:61]: Nice fix!

**Review by kholdaway — APPROVED** [review]

**Pull request merged**